site/content/entry/yuzu-progress-report-sep-2020/index.md

```diff
@@ -0,0 +1,133 @@
+
+While this should work with games such as `Super Mario Odyssey` and others, do note it won't be able to cover all cases or titles. Every game is different, and the required input is not always just a shake.
+
+## Remember the Rumble Pak?
+
+Also thanks to [german77](https://github.com/german77), yuzu can now proudly boast [Rumble support](https://github.com/yuzu-emu/yuzu/pull/4291). Any SDL2 compatible device will vibrate when asked by your Switch games. This includes but is not limited to DualShock, Xbox, and Nintendo controllers.
+
+HD Rumble is not possible, yet. The SDL2 specification is incompatible, and not everyone will play with Nintendo peripherals. This means that a Pro Controller or a pair of Joy-Cons cannot reach the same level of precision as the Switch. Additionally, some games seem to use different ways to ask for rumble and are not supported yet. We are investigating this.
+
+Soon, german77 has plans to add native Joy-Con pairing support. This will allow for real HD Rumble compatibility.
+
+## Video decoding, or how to suffer for so long with a single codec
+
+It's finally here! After such a long wait, and thanks to the work from [ogniK](https://github.com/ogniK5377) and [epicboy](https://github.com/ameerj), [yuzu can now play videos in-game!](https://github.com/yuzu-emu/yuzu/pull/4729) For now, `H.264` and `VP9` videos are supported, which thankfully seems to cover all released games until now. Support for `H.265` and `VP8` will come later.
+
+{{< imgs
+    "./mk8.mp4| Video Killed the Radio Star (Mario Kart 8 Deluxe)"
+  >}}
+  
+This is possible by using [FFmpeg.](https://ffmpeg.org/) For now, decoding happens in a single CPU thread, and support for multiple threads and GPU decoding is currently being worked on.
+
+We have an entire article dedicated to showing how this feature came to be, how it works, its challenges, and current limitations in more detail [here.](https://yuzu-emu.org/entry/yuzu-nvdec-emulation/) Please take your time to read it: it also includes a list of games confirmed working and not-so-working yet.
+
+## An emulator inside an emulator
+
+`Super Mario 3D All-Stars` is a special case, in several aspects. For starters, the game is just a container for several other binary executables (known as `Program NCAs`), each one with their own Title ID. Step one to get this game to boot is to handle [a particular case like this](https://github.com/yuzu-emu/yuzu/pull/4675), a job [Morph](https://github.com/Morph1984) did. 
```

Comment:
Change `to handle a particular case like this` to `to correctly handle these multi-content games`